ui/components/multichain/receive-modal/__snapshots__/receive-modal.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ReceiveModal should show the correct account address and name 1`] = `<div />`;
```

Comment:
Jest should render the component tree. wdyt?

Reply:
I'll remove this snapshot, it doesn't have value.